package.json

```diff
@@ -34,6 +34,7 @@
     "@angular/platform-browser": "2.3.0",
     "babel-cli": "^6.18.0",
     "babel-plugin-transform-react-jsx": "^6.8.0",
+    "babel-preset-es2015-rollup": "^3.0.0",
```

Comment:
@Rich-Harris Unrelated to the pr itself maybe we should make another preset that includes external helpers?

currently babel-preset-es2015 allows you to choose the modules plugin: http://babeljs.io/docs/plugins/preset-es2015/#options with `modules: false` to not compile es modules.

es2015 rollup is just `modules: false` + `external-helpers`. We could just add it to preset-env?